db/migrate/20160309182642_remove_social_events_table.rb

```diff
@@ -0,0 +1,134 @@
+class RemoveSocialEventsTable < ActiveRecord::Migration
```

Comment:
> Would that be sth like f9bf080

Yes exactly.
> Would we have to invoke the rake task from within the migration

This is the thing. Data migration is part of the deployment, not part of the development. So for us, we would add that new rake task to `config/deploy.rb` and for other deployments we would have to come up with a way to describe how to do this.

This also means that dropping the old schema should not happen in the same commit.